src/components/Reports/Reports.js

```diff
@@ -16,7 +16,7 @@ import { Loading } from '../Loading/Loading';
 import { addDays, getStartOfDate } from '../../utils';
 
 // This value means the today date
-const periodSelectedDaysDefault = 0;
+const periodSelectedDaysDefault = 7;
```

Comment:
With this the default is one week, is it intended?

Reply:
@GusBaamonde told me it's intended.

Reply:
yes, maybe need change the comment